### DISCUSSION TITLE
Is there a way to not show empty workspaces in SwayWorkspaces?

### DISCUSSION THREAD

**Discussion by ek9** [nwg-piotr/nwg-shell#59] [Q&A] [answered]
Is there a way to not show empty workspaces in the list? Currently the ones with windows are being marked with a `.` by default. I'm trying to figure out a way to only display workstation numbers if they have any windows?

Current:
![workspaces all](https://imgur.com/BYthz18.png)

Desired:
![show only non empty workspaces](https://imgur.com/c1m6Bzj.png)

Any pointers appreciated

**Comment by nwg-piotr** [comment]
Well, no. In real life I prefer the SwayTaskbar module, so I'm not really focused on the SwayWorkspaces development. It would be better if someone who really uses it could contribute.

**Comment by aposteeltje-bwg** [comment, marked as answer]
Mark 'hide empty workspaces' ?
![swappy-20240104_202819](https://github.com/nwg-piotr/nwg-shell/assets/155484093/a3444d7c-f8a9-4adf-ae30-c1308783bad4)

**Reply by nwg-piotr** [reply]
Yes. This thread was heavily out of date.
